### PR TITLE
Detailed page

### PR DESCRIPTION
**Pull request by SimonGideon**
## Description
When the user selects a specific item, they can see the details page with its full description. On the details page, the user can click the "Reserve" button.
## Checklist
- [x] When the user selects a specific item, they can see the details page with its full description. On the details page, the user can click the "Reserve" button.

## Additional Notes
Thanks for the review

### CODE REVIEW

**Review by enoqueJonas — CHANGES_REQUESTED** [review]
[STATUS] Changes Required ♻️
Hi @SimonGideon,

# Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Organized code

## Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/components/Detail.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <button type="submit" className="myButton">
+            {' '}
+            <FaCog />
+            {' '}
+            Reserve
+            <span className="circle-icon">
+              <FiChevronRight />
+            </span>
+            {' '}
+          </button>
```
Comment (enoqueJonas) [inline src/components/Detail.jsx]: When clicking the Reserve button, the room is supposed to be reserved. Kindly add an event that will change the reserved attribute to true.

Comment (enoqueJonas) [inline src/components/Detail.jsx]: [Optional] Your button follows the design guidelines but it only shows the green color when the mouse hovers over the button. Kindly make the green color the default color of the button.

Comment (SimonGideon) [reply src/components/Detail.jsx]: There is a card for adding reservations. It will be implemented on another pr

**Review by enoqueJonas — APPROVED** [review]
Hi @SimonGideon, there's nothing else for me to say other than… Your project is complete, it's time to merge it🥳

# Status: Approved✅

Your project is complete.

 ## Highlights
- Implemented requested changes

## Optional suggestions
Every comment with [optional] prefix won't stop the approval of the PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and happy coding👨🏾‍💻

*Feel free to leave any questions in the comments in the PR thread if something is not 100% clear. Please remember to tag me so I can receive the notification.*

**Pull request merged**